src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -393,7 +393,7 @@ void AbstractTableGenerator::_create_table_indexes(
       Assert(index_column_names.size() == 1, "Multi-column indexes are currently not supported.");
 
       for (const auto& column_name : index_column_names) {
-        std::cout << "-  Creating an index on table " << table_name << std::flush;
+        std::cout << "-  Creating secondary index on '" << table_name << "." << column_name << "'" << std::flush;
```

Comment:
Aren't most (or even all?) indexed we define primary indexes?

Reply:
A primary index is a sorted table. We never do that.

Reply:
I looked it up and it seems that different definitions disagree on this. Some just rquire primary indexes to be on the primary key columns 
> A unique index on the primary key of the table. [[IBM DB2 doc](https://www.ibm.com/docs/en/db2-for-zos/12?topic=idi-types-indexes)]

and some require it to determine the position of the tuple
> A primary index determines the location of the records of the data file, while a secondary index does not. [Garcia-Molina et al. Database Systems.
The Complete Book. 2nd edition, p. 620.]

So just index?